rbac/server/api/proposals.py

```diff
@@ -168,6 +168,7 @@
     await utils.send(
         request.app.config.VAL_CONN, batch_list, request.app.config.TIMEOUT
     )
+    await utils.send_notification(proposal_resource.get("target"), proposal_id)
```

Comment:
When we approve/reject UpdateUserManager proposals, do we want to send the notification to the manager or the user? For this proposal, I believe the object_id is the user that is having their manager changed and the related_id is the new manager.

Reply:
We want to send this to the manager.  Once it is approved and updated it will alert the user to the change.  Which is taken care of under update proposal.  The "target" is the new manger/ approver according to the database entry

Reply:
Sounds good.